src/colmap/sfm/observation_manager.h

```diff
@@ -51,9 +51,9 @@ class ObservationManager {
     size_t num_total_corrs = 0;
   };
 
-  ObservationManager(std::shared_ptr<Reconstruction> reconstruction,
-                     std::shared_ptr<const CorrespondenceGraph>
-                         correspondence_graph = nullptr);
+  explicit ObservationManager(std::shared_ptr<Reconstruction> reconstruction,
```

Comment:
nit: explicit only meaningful for single-arg constructor.

Reply:
Done.

Reply:
ClangTidy actually requires it because the second argument is optional.